r2-core/src/main/java/com/linkedin/r2/filter/TimedStreamFilter.java

```diff
@@ -34,7 +36,7 @@
  *
  * @author Xialin Zhu
  */
-/* package private */ class TimedStreamFilter implements StreamFilter
+public class TimedStreamFilter implements StreamFilter
```

Comment:
Curios, why did we made this public?

Reply:
We need to call the methods in another package.